docker/Dockerfile

```diff
@@ -13,7 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ##
-FROM aivenoy/kafka:3.3-2022-10-06-tiered-storage-1
+# TODO: fix once `aivenoy/` image is available on docker hub
+FROM jeqo/kafka:kips_KIP-405_2023-08-18
```

Comment:
just for testing purposes, I used an image from my own repo, and CI is passing. We can switch to `aivenoy` image when available.